backend/src/database/state-migrations/impossible-migration.ts

```diff
@@ -0,0 +1,20 @@
+import { RestoreError } from '../../utils/restore-error';
+import type { MigrationFunctions } from './migrations';
+
+export const impossibleMigration: MigrationFunctions = {
```

Comment:
I would rename all the variables, classes, interfaces, etc. from `...MigrationFunctions` to `...Migrations`